Themes/default/css/index.css

```diff
@@ -57,6 +58,10 @@ textarea, textarea.editor {
 input, button, select {
 	font-size: 0.9em;
 }
+/* Height fixation */
+input, select {
```

Comment:
May I know the reason for introduction yet another CSS class?

Why not just use :not? http://caniuse.com/#feat=css-sel3